### DISCUSSION TITLE
how do i add freeplay categories 🙏🙏🙏

### DISCUSSION THREAD

**Discussion by m4ttzzzz** [CodenameCrew/CodenameEngine#453] [Q&A]
how to add freeplay categories? (im making a mod that is a collection of songs that i've charted and i want to categorize them by game)

**Comment by THEkatinamicrowave** [comment]
Steps to do this: make a FPCatSubState in the states folder and then override FreeplayState with your own category selector. I'm sure you can probably just steal it from the OptionsScreen.hx file. If needbe I'll whip up some files
